Gemfile

```diff
@@ -51,6 +51,8 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'bullet'
+  gem 'database_cleaner'
   gem 'debug', platforms: %i[mri mingw x64_mingw]
 end

```

Comment:
- [ ] Please make sure that you have the `Capybara` gem in `Gemfile` for the integration test :pray:

Reply:
All done...
All test cases passed

Reply:
`gem 'capybara'` exist